src/seedu/addressbook/Main.java

```diff
@@ -20,6 +24,10 @@
  */
 public class Main {
 
+    private static final String TAGSESSIONHISTORY_HEADER = "Tags added/removed during session";
+    private static final String TAGSESSIONHISTORY_REMOVE_ACTION_SYMBOL = "-";
+    private static final String TAGSESSIONHISTORY_ADD_ACTION_SYMBOL = "+";
```

Comment:
The constant names are too long right? Especially `TAGSESSIONHISTORY` is not really that readable.
Perhaps these constants could be refactored some where else? \
Perhaps you could have a `SessionHistory` class to keep track of the entire session history, be it tags or people added?
In this way the constant names could be reduced to `TAG_HEADER` in the `SessionHistory` class perhaps?